### DISCUSSION TITLE
🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵🎵_begging for your favorite songs

### DISCUSSION THREAD

**Discussion by WEI44ZHEYUAN** [NAIST-SE/AttractingContributors#860] [2023 Attraction Contributions]
Please let me know you favorite songs , Thanks，ありがとう，Danke，감사해요，Спасибо，Merci，Gracias，ขอบคุณ，شكرًا，Grazie，謝謝，谢谢

https://github.com/WEI44ZHEYUAN/AttractingContributors_Songs

**Comment by TakumaMiwa** [comment]
I made a pull request! Please help me!
https://github.com/TakumaMiwa/AttractingContributors-Music

**Comment by ALORAMFER** [comment]
Can you help me please? haha
https://github.com/ALORAMFER/AttractingContributors-SoccerTeams
